staging-service-shared/src/main/java/com/github/cafdataprocessing/services/staging/dao/filesystem/BatchNameProvider.java

```diff
@@ -39,7 +41,9 @@ public static String getSubBatchName()
 
     private static String getCurrentTimeAsString()
     {
-        return cleanseTime(Instant.now().toString());
+        final DateTimeFormatter formatToday = DateTimeFormatter.ofPattern(DATE_TIME_ISO_PATTERN).withZone(ZoneOffset.UTC);
+        final String currentTimeAsString = formatToday.format(Instant.now());
+        return cleanseTime(currentTimeAsString);
```

Comment:
Still using `cleanseTime()` doesn't make much sense in the light of the other changes being made.